test/tests/citeTest.js

```diff
@@ -0,0 +1,48 @@
+describe("Zotero.Cite.System", function() {
+	describe("#retrieveItem()", function() {
+		let article, citeSystem;
+		before(function() {
+			// Load sample item into DB
+			let data = populateDBWithSampleData(loadSampleData('journalArticle'));
+			article = Zotero.Items.get(data.journalArticle.id);
+			
+			// instantiate Zotero.Cite.System object
+			citeSystem = new Zotero.Cite.System();
+		});
+		
+		it('should retrieve item given a Zotero.Item', function() {
+			let cslItem = citeSystem.retrieveItem(article);
+			assert.ok(cslItem);
+			assert.equal(cslItem.id, article.id);
+		});
+		
+		it.skip('should retrieve item given an item URI', function() {
```

Comment:
Any thoughts on how to test it? Doesn't sound like it would be very simple. Also, I think this test is still something we would want to work in the future (no?), so I'll leave it in.